Markdown checker failure, reported upstream

The markdown checker fails on the https://jwst-docs.stsci.edu/ link, but the link is fine.  I've reported this false positive in https://github.com/UmbrellaDocs/linkspector/issues/167

The failure:
```
NIRCam/README.md:12:1: 🚫 https://jwst-docs.stsci.edu/jwst-near-infrared-camera/nircam-instrumentation/nircam-detector-overview/nircam-detector-performance#gsc.tab=0 Status:null net::ERR_EMPTY_RESPONSE at https://jwst-docs.stsci.edu/jwst-near-infrared-camera/nircam-instrumentation/nircam-detector-overview/nircam-detector-performance#gsc.tab=0
- Checking NIRCam/README.md...
💥 Error: Some hyperlinks in the specified files are invalid.
Error: Process completed with exit code 1.
```

--- a/.github/workflows/markdown_link_check.yml
+++ b/.github/workflows/markdown_link_check.yml
@@ -24,6 +24,8 @@ jobs:
     - name: Check links
       run: |
         # Generate default configuration file if it doesn't exist.
-        if [ ! -f .linkspector.yml ] ; then printf "dirs:\n  - ./\nignorePatterns:\n  - pattern: '^https://www.gnu.org/licenses.*$'\n" > .linkspector.yml ; fi
+        # - github.com URL's can lead to rate limiting errors.
+        # - stsci.edu URL's can lead to connection refused errors.
+        if [ ! -f .linkspector.yml ] ; then printf "dirs:\n  - ./\nignorePatterns:\n  - pattern: '^https://www.gnu.org/licenses.*$'\n  - pattern: '^https://jwst-docs.stsci.edu/$'\n" > .linkspector.yml ; fi
         # Run the check.
         linkspector check

--- a/README.md
+++ b/README.md
@@ -12,3 +12,10 @@ This quote highlights the goal of this repository:
 [arghwhat](https://news.ycombinator.com/item?id=36354464)
 
 > The problem with DevOps is not really how it is practiced, but how it is named. DevOps is a bucket that a bunch of things are thrown into, and everybody thinks the thing they threw in is DevOps with the others being mislabelled.
+
+## URL's to test the markdown linkchecker action
+
+These URL's have caused trouble with the markdown link checker, and are
+therefore included here:
+
+- https://jwst-docs.stsci.edu/

